packages/react-composites/src/composites/CallComposite/utils/Utils.ts

```diff
@@ -60,6 +62,21 @@ enum CallEndReasons {
 }
 
 const getCallEndReason = (call: CallState): CallEndReasons => {
+  const remoteParticipantsEndedArray = Array.from(Object.values(call.remoteParticipantsEnded));
+  /**
+   * Handle the special case in a PSTN call where removing the last user kicks the caller out of the call.
```

Comment:
Tested with Anjul. Yes, that is the case. This will only work in an initialized 1:1 call scenario without any other participant being added to the call.